src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -142,6 +151,7 @@ public BitbucketCommit call() throws Exception {
     private static final String V2_API_BASE_URL = "https://api.bitbucket.org/2.0/repositories";
     private static final String V2_TEAMS_API_BASE_URL = "https://api.bitbucket.org/2.0/teams";
     private static final String REPO_URL_TEMPLATE = V2_API_BASE_URL + "{/owner,repo}";
+    private static final String AVATAR_URL = BitbucketCloudEndpoint.SERVER_URL+ "rest/api/1.0/{/owner}/projects/{repo}/avatar.png";
```

Comment:
https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D#get

which should have a "avatar.href"

Reply:
I have not seriously looked at the Cloud yet, but I thought I copied the URL that was in the project already. Thanks for the link

Reply:
https://confluence.atlassian.com/bitbucket/version-1-423626337.html
>Bitbucket Cloud v1 APIs are deprecated
>Bitbucket Cloud REST API version 1 is deprecated effective 30 June 2018, and were removed from the REST API permanently on 29 April 2019. Read the deprecation notice. Or you can jump right to the version 2.0 REST API documentation.

Please look into v2 endpoint for the avatar url 🤔